src/js/flows/removeConnection.ts

```diff
@@ -0,0 +1,29 @@
+import Clusters from "../state/Clusters"
+import {Thunk} from "../state/types"
+import {popNotice} from "../components/PopNotice"
+import {Cluster} from "../state/Clusters/types"
+import ConnectionStatuses from "../state/ConnectionStatuses"
+import Current from "../state/Current"
+import Spaces from "../state/Spaces"
+import Investigation from "../state/Investigation"
+
+const removeConnection = (conn: Cluster): Thunk => (
+  dispatch,
+  _getState,
+  {globalDispatch}
+) => {
+  const {name, id, host, port} = conn
+
+  if (host === "localhost" && port === "9867")
+    throw new Error("Cannot remove the default connection")
```

Comment:
I like having this check here. Let's also remove the remove (haha) button from the page if we're looking at the details for localhost:9867.